xml/System.Threading.Tasks/Task.xml

```diff
@@ -2640,7 +2640,7 @@ Task t Status: RanToCompletion, Result: 42
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- A <xref:System.Threading.Tasks.Task> will complete in the <xref:System.Threading.Tasks.TaskStatus?displayProperty=fullName> state under any of the following conditions:  
+ A <xref:System.Threading.Tasks.Task> will complete in the <xref:System.Threading.Tasks.TaskStatus.Canceled> state under any of the following conditions:  
```

Comment:
The xref should be `<xref:System.Threading.Tasks.TaskStatus.Canceled?displayProperty=fullName>`.